heuristics/dbic_bids.py

```diff
@@ -613,9 +617,11 @@ def infotoids(seqinfos, outdir):
     }
 
 
+_SANITIZER = maketrans('', '', '#!@$%^&.,:;_-')
```

Comment:
Never used this one, cool... Quick search showed that it takes to args not 3 though